app/components/main-content/PrelaunchSignUpForm.tsx

```diff
@@ -38,13 +37,22 @@ const PrelaunchSignUpForm: React.FC<PrelaunchSignUpFormProps> = ({
   async function handleSignupSubmit(rawUserData: UserDataValidation) {
     setIsLoading(true);
 
-    const sanitizedUserData = userDataSanitizationSchema.parse(rawUserData);
+    const { email, name, interests, source } = rawUserData;
+
+    const stringifiedInterests = JSON.stringify(interests);
+
+    window.localStorage.setItem('userName', name);
+    if (interests) {
+      window.localStorage.setItem('userInterest', stringifiedInterests);
+    }
+    if (source) {
+      window.localStorage.setItem('userSource', source);
+    }
 
     try {
-      const userId = sanitizedUserData.email; // use user email as user Id to ensure user uniqueness
-      await storeUserDataIfNew(userId, sanitizedUserData);
+      sendEmailLink(email);
```

Comment:
The current validation logic does not remove white spaces. We will need to sanitize the email before passing it to the sendEmailLink function to ensure the email is valid.

Reply:
Since the validation schema handles everything except white spaces, I think the most straightforward fix here is adding `.trim()` to the argument we're passing to `sendEmailLink`:

``` typescript
sendEmailLink(email.trim())
```

We could use `userDataSanitizationSchema` to parse the email but as it's currently written, I believe we would need to also provide a value for `name` to the schema. 

What do you think of just using `.trim()` here?

Reply:
trim is commonly used and more performant than calling zod - it is a good choice for this use case